.github/workflows/ci.yml

```diff
@@ -1,90 +1,26 @@
-# ------------------------------------------------------------------------------
-# <auto-generated>
-#
-#     This code was generated.
-#
-#     - To turn off auto-generation set:
-#
-#         [GitHubActions (AutoGenerate = false)]
-#
-#     - To trigger manual generation invoke:
-#
-#         nuke --generate-configuration GitHubActions_ci --host GitHubActions
-#
-# </auto-generated>
-# ------------------------------------------------------------------------------
-
 name: ci
 
 on:
   push:
-    branches:
-      - main
-      - 'refs/tags/*'
-    paths:
-      - '!docs/*'
```

Comment:
Did you mean to remove the docs exclusion? It seems useful to exclude docs-only changes (unless some checks are being enforced and therefore they would still have to run)

Reply:
> unless some checks are being enforced 

In "legacy main" the builds were required. Now there are not, but I am not sure if this is as expected.

If we do not want to require any builds then I can bring it back.

Reply:
There's no harm in leaving this change. If we feel like it slows down development, we can always add the exclusion back in later